tests/conftest.py

```diff
@@ -116,18 +116,24 @@ def client(request):
     if marker:
         setup_params.update(marker.kwargs)
 
+    # passphrase is turned on either for True or the str directly
+    use_passphrase = setup_params["passphrase"] not in (None, False)
```

Comment:
this is somewhat confusing, I'd prefer the inverse check?
`use_passphrase = setup_params["passphrase"] is True or isinstance(setup_params["passphrase"], str)`

Reply:
if we replace all usages of `passphrase=True` with strings, this could be simpler too.

i'm not sure if that is possible everywhere, but perhaps tests that need passphrase enabled but not prefilled can do that explicitly?

Reply:
Sounds good! Moved something to #1110 as well.

Reply:
The rest in ac9f57fa8e1406efa6360fdb53ace5962940be1e